core/platform/app_identity/gae_app_identity_services.py

```
@@ -34,10 +34,13 @@ def get_application_id():
     Returns:
         str. The application ID.
     """
-    return os.getenv('GOOGLE_CLOUD_PROJECT')
+    app_id = os.getenv('GOOGLE_CLOUD_PROJECT')
+    # Ruling out the possibility of None for mypy type checking.
+    assert app_id is not None
```

Hmm, not sure if we want to do this here, or rather throw a proper error (no assertion error) if it is not set.

---

Done

---

But line 39 here is not covered by backend tests. How should I proceed? Thanks!

---

Add a backend test for it then. Also, do we still need this line?

---

Done. Removed the assert too.